lambdaclass/mirra_backend
# [GH-775] [GH-741] Add login and homepage to Configurator

## Motivation

Its hard to tune the settings because we have to remember every path.
We need a way to restrict users to change our game settings.
Closes #775
Closes #741

## Summary of changes

- [Add everything related to user login and authentication. Request them for every settings route](https://github.com/lambdaclass/mirra_backend/commit/cbcb668c152113a2ae165785f91853341b31b87b)
- [Add tests for user authentication modules](https://github.com/lambdaclass/mirra_backend/commit/42903c116deebbbbf57b22f4802b55f4f0f3f834)
- [Add Configurator landing page aka home](https://github.com/lambdaclass/mirra_backend/commit/ea4893b584ca62cba73a6ec5ab8af96ebcc5fbf8)
- [Add styles to every rendered template in Configurator](https://github.com/lambdaclass/mirra_backend/commit/19518d2762cbe41be2e035142b87ff28bec9f285)

## How to test it?

Run the application.
Enter `localhost:4100` to see the homepage.
You shouldn't be able to enter any route without signing in.
There's a user for local testing purposes in `seeds.ex`.

## Checklist
